test/e2e_closure_test.ts

```diff
@@ -26,6 +26,7 @@ describe('golden file tests', () => {
     goldenJs.push('third_party/tslib/externs.js');
     goldenJs.push('third_party/tslib/tslib.js');
     goldenJs.push('test_files/augment/shim.js');
+    goldenJs.push('test_files/clutz.no_externs/default_export.js');
```

Comment:
Why don't we need to push `import_default.js` here?

Reply:
The line above first collects all `.js` files that are generated from a `.ts` file. The files manually added here are the ones not generated from a `.ts` file, i.e. files that contain the definition of a symbol that's only declared (i.e. in a `.d.ts` file) in TS land.

I've added comments explaining that.